DataFormats/SiPixelDigiSoA/interface/alpaka/SiPixelDigiErrorsDevice.h

```diff
@@ -0,0 +1,75 @@
+#ifndef AlpakaDataFormats_alpaka_SiPixelDigiErrorsAlpaka_h
+#define AlpakaDataFormats_alpaka_SiPixelDigiErrorsAlpaka_h
+
+#include <cstdint>
+#include <alpaka/alpaka.hpp>
+#include "DataFormats/SiPixelDigiSoA/interface/SiPixelDigiErrorsHost.h"
+#include "DataFormats/SiPixelRawData/interface/SiPixelErrorCompact.h"
+#include "DataFormats/SiPixelRawData/interface/SiPixelFormatterErrors.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "HeterogeneousCore/AlpakaInterface/interface/SimpleVector.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/CopyToHost.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class SiPixelDigiErrorsDevice {
```

Comment:
Needs type alias for `alpaka_serial_sync::SiPixelDigiErrorsDevice = SiPixelDigiErrorsHost`